README.md

```diff
@@ -64,6 +64,17 @@ podman_services:
     # Optional: Define a hostname for podman's hostname flag. Set's
     # the containers hostname, default is none.
     hostname: chasinglogic.io
+    # Optional: A list of secrets to mount. Secrets will not be created
+    # automatically. The only required field if used is name. The other
+    # values have defaults as specified in podman documentation
```

Comment:
just slap this link on the end of the comment please: https://docs.podman.io/en/v4.6.0/markdown/options/secret.html